docs/src/_cloud/onboarding.md

```diff
@@ -93,32 +69,66 @@ pipx reinstall meltano-cloud-cli
 
 The above commands will install a `meltano-cloud` CLI command into your workstation.
 
-### Step 3: User creation and authorization
+### Step 2: Login to Meltano Cloud
 
-First, create your user account by logging in via: `meltano-cloud login`.
+Login to Meltano Cloud by running `meltano-cloud login` in your local terminal.
```

Comment:
```suggestion
Login to Meltano Cloud by running the `meltano-cloud login` [command](https://docs.meltano.com/cloud/cloud-cli#login) in your local terminal.
```

@cjohnhanson what do you think about adding CLI command reference links to each of these so they can dive deeper if needed.

Reply:
@pnadolny13 I think that's a really good idea. Thanks for adding the suggested changes, I'll go through and double check any remaining commands and add the links.